test/samlp.tests.js

```diff
@@ -7,59 +7,6 @@ var Samlp   = require('../lib/passport-wsfed-saml2/samlp');
 var Saml    = require('../lib/passport-wsfed-saml2/saml').SAML;
 
 describe('samlp (unit tests)', function () {
-  describe('samlrequest with template', function() {
```

Comment:
The two tests inside actually belong to the `describe()` for `getSamlRequestParams()`, which is why I moved them (see below).